src/main/java/uk/gov/ons/ctp/common/jackson/CustomObjectMapper.java

```diff
@@ -3,14 +3,14 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
-import java.text.SimpleDateFormat;
+import uk.gov.ons.ctp.common.util.MultiIsoDateFormat;
 
 /** Custom Object Mapper */
 public class CustomObjectMapper extends ObjectMapper {
 
   /** Custom Object Mapper Constructor */
   public CustomObjectMapper() {
-    this.setDateFormat(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX"));
```

Comment:
woops how did we miss that 🤦‍♂️  . Is it possible to rebuild the artefact with the same version (change version to 10.49.17-SNAPSHOT and delete 10.49.18 from artifactory??) saves us going over and updating all the other services again...

Reply:
That's quite a good idea. Depends if we can delete from Artifactory or not